daemon/cmd/status_test.go

```diff
@@ -37,18 +32,18 @@ var (
 	}
 )
 
-func (g *GetNodesSuite) SetUpTest(c *C) {
+func setup(t *testing.T) {
+	var err error
```

Comment:
Add `t.Helper()` call here to mark it as a test helper functions and so failures will be attributed to the caller.

```suggestion
func setup(t *testing.T) {
	t.Helper()
	
	var err error
```